ext/nnstreamer/tensor_decoder/tensordec-directvideo.c

```diff
@@ -174,6 +174,13 @@ dv_getOutCaps (void **pdata, const GstTensorsConfig * config)
   gint width, height, channel;
   GstCaps *caps;
 
+  if (config->info.info[0].type != _NNS_UINT8) {
```

Comment:
direct_video also support `_NNS_UINT16`

Reply:
fixed to take care of element size consistency only.